server/app/views/applicant/EditTiClientView.java

```diff
@@ -0,0 +1,227 @@
+        && !form.get().errors(TrustedIntermediaryService.FORM_FIELD_NAME_PHONE).isEmpty()) {
+      phoneNumberField.setFieldErrors(
+          messages, form.get().errors(TrustedIntermediaryService.FORM_FIELD_NAME_PHONE));
+    }
+    FieldWithLabel emailField =
+        FieldWithLabel.email()
+            .setId("edit-email-input")
+            .setFieldName("emailAddress")
+            .setLabelText("Email Address")
+            .setToolTipIcon(Icons.INFO)
+            .setToolTipText(
+                "Add an email address for your client to receive status updates about their"
+                    + " application automatically. Without an email, you or your community-based"
+                    + " organization will be responsible for communicating updates to your"
+                    + " client.")
+            .setValue(account.getEmailAddress());
+    if (form.isPresent()
+        && form.get().hasErrors()
+        && !form.get().errors(TrustedIntermediaryService.FORM_FIELD_NAME_EMAIL_ADDRESS).isEmpty()) {
+      emailField.setFieldErrors(
+          messages, form.get().errors(TrustedIntermediaryService.FORM_FIELD_NAME_EMAIL_ADDRESS));
+    }
+    FieldWithLabel dateOfBirthField =
+        FieldWithLabel.date()
+            .setId("edit-date-of-birth-input")
+            .setFieldName("dob")
+            .setLabelText("Date Of Birth")
+            .setRequired(true)
+            .setValue(
+                applicantData
+                    .getDateOfBirth()
+                    .map(this.dateConverter::formatIso8601Date)
+                    .orElse(""));
+    if (form.isPresent()
+        && form.get().hasErrors()
+        && !form.get().errors(TrustedIntermediaryService.FORM_FIELD_NAME_DOB).isEmpty()) {
+      dateOfBirthField.setFieldErrors(
+          messages, form.get().errors(TrustedIntermediaryService.FORM_FIELD_NAME_DOB));
+    }
+    FieldWithLabel tiNoteField =
+        FieldWithLabel.textArea()
+            .setId("edit-ti-note-input")
+            .setFieldName("tiNote")
+            .setLabelText("Notes")
+            .setValue(account.getTiNote());
+    String cancelUrl =
+        baseUrl
+            + controllers.ti.routes.TrustedIntermediaryController.dashboard(
+                    /* nameQuery= */ Optional.empty(),
+                    /* dateQuery= */ Optional.empty(),
+                    /* page= */ Optional.of(1))
+                .url();
+    return div()
+        .with(
+            formTag
+                .with(
+                    firstNameField.getInputTag(),
+                    middleNameField.getInputTag(),
+                    lastNameField.getInputTag(),
+                    phoneNumberField.getInputTag(),
+                    emailField.getEmailTag(),
+                    dateOfBirthField.getDateTag(),
+                    tiNoteField.getTextareaTag(),
+                    makeCsrfTokenInputTag(request),
+                    submitButton("Save").withClasses("ml-2", "mb-6"),
```

Comment:
Sijia gave this Design. The mocks are attached to the githib ticket-https://github.com/civiform/civiform/issues/4956

Reply:
Ah I see. I think with a modal that flow makes more sense. But now that it's a separate screen, it's a little confusing. I see below that the "Successfully updated client" success toast is showing for you, so let me try that out because that might be enough to make it clear.

Reply:
I think it's clear enough with the toast message. I'll resolve this.